content/gophercon-2017/writing-a-successful-gophercon-proposal.md

```diff
@@ -0,0 +1,100 @@
++++
+author = ["David Cheney"]
+date = "2017-01-14"
+title = "Writing a successful GopherCon proposal"
+tags = ["gophercon","community"]
+series = ["GopherCon 2017"]
+
++++
+
+[The GopherCon 2017 Call for Proposals](https://www.papercall.io/gophercon2017) has reached the halfway mark so I wanted to pass on some advice to potential speakers who are planning to submit proposals but may be unsure of how to go about it.
+
+# Consider your audience.
+
+This year we're asking for proposals for three different kinds of talks; keynotes, tutorials, and workshops.
+The audience for each of these are different.
+
+For keynotes, you'll be speaking to the entire audience, 1500 people whose experience with Go will span everything from beginners, to long time GopherCon attendees, to the Go team themselves.
+This means you have to keep your topic relevant for a wide audience.
+
+But, this does not mean pitching to the loweset common denominator and trying to sell the audience on the benefits of using Go.
+The audience have already invested time and money in flying to Denver to visit the conference -- they already think Go is pretty great.
+Instead, you want to inspire the audience to convince _other_ people that Go is great, and do that by presenting information that is relevant to Go programmers today.
+
+For tutorials, you have more time, up to 45 minutes, and an audience who are self selecting; they have chosen to come to your talk over the other two talks at the same time.
+You can afford to present in depth on a topic and be prepared to take questions.
+
+For workshops, again you have more time, the whole day, to instruct students who have chosen to attend your specific workshop.
```

Comment:
Perhaps let's change this to:
to instruct a small group of people who want specialized knowledge